waspc/examples/todo-typescript/package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "prototype",
   "dependencies": {
-    "wasp": "file:.wasp/out/sdk/wasp",
-    "react": "^18.2.0"
+    "cors": "^2.8.5",
```

Comment:
Following up on our discussion today, I didn't end up adding `cors` to list of deps in SDK. I was considering it, but realized `cors` is not part of our API. Yes, we do use it in our middleware, but you can provide any middleware you want. By "accident", that can also be cors middleware, and in that case you will write over our cors middleware. But you don't have to use that `cors` library, you can use your own library for cors, or `cors-something` library, or whatever. So it is not part of our API, and this `cors` here is "random".
This makes the situation very clear, so this is the right choice.